openedx/core/djangoapps/user_api/learner_skill_levels/api.py

```diff
@@ -0,0 +1,120 @@
+"""
+APIs for learner skill levels.
+"""
+from opaque_keys.edx.locator import CourseLocator
+
+from lms.djangoapps.grades.models import PersistentCourseGrade  # lint-amnesty, pylint: disable=unused-import
+from openedx.core.djangoapps.catalog.utils import get_course_data
+
+from .constants import LEVEL_TYPE_SCORE_MAPPING
+
+
+def _calculate_user_skill_score(all_skills):
+    """
+    Calculates user skill score to see where the user falls in a certain job category.
+    """
+    sum_of_skills = sum(all_skills.values())
+    skills_count = len(all_skills)
+    # sum of skills score in the category/ 3*no. of skills in category
+    if skills_count:
+        return round(sum_of_skills / (3 * skills_count), 1)
+    return 0.0
+
+
+def _get_category_skill_score(category_skills, learner_skill_score):
```

Comment:
if will be better if keep the name generic . so instead of `category_skills ` just skills.